kimchi_description/README.md

```diff
@@ -0,0 +1,26 @@
+# kimchi_description
+
+## Description
+This package holds the urdf description of the robot.
+
+<img src="docs/robot_rviz.png">
```

Comment:
This pkg doesn't have a "docs" dir, so I would expect this image to be broken. Would it make sense to add another image?